WakaTime.py

```diff
@@ -384,7 +384,7 @@ def process_queue(timestamp):
 
     # Prevent sending heartbeats more often than SEND_BUFFER_SECONDS
     now = int(time.time())
-    if timestamp != LAST_HEARTBEAT['time'] and LAST_HEARTBEAT_SENT_AT > now - SEND_BUFFER_SECONDS:
+    if isCliInstalled() and timestamp != LAST_HEARTBEAT['time'] and LAST_HEARTBEAT_SENT_AT > now - SEND_BUFFER_SECONDS:
```

Comment:
No, that needs to be an `or`. I'll make the fix on `master` branch.

Reply:
tks